keyboards/massdrop/thekey/DOCUMENTATION.md

```diff
@@ -0,0 +1 @@
+to be written
```

Comment:
This file should be removed.

Reply:
removed

Reply:
Looks like it's still there. 

If you're using CLI, `git rm (filename)` and then committing will fix the issue. 

Otherwise, you can delete the file from the web interface.

Reply:
Yep - just removed it on the wrong branch like a fool